win32.cmake

```diff
@@ -6,7 +6,7 @@ set(TEE_SOURCES
     src/Windows/metee_winhelpers.c
 )
 
-add_library(${PROJECT_NAME} ${TEE_SOURCES})
+add_library(${PROJECT_NAME} SHARED ${TEE_SOURCES})
```

Comment:
This is used statically in LMS and other projects, maybe you want to create both libraries....

Reply:
No need for that, there is BUILD_SHARED_LIBS configuration flags